### PR TITLE
Release notes for `4.20.0`

### PR DESCRIPTION
**Pull request by NachoSoto**
From #2543.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,35 @@
-## 4.18.1
```
Comment (NachoSoto) [inline CHANGELOG.md:1]: This was wrong in #2491 (it was a patch initially before made a minor release).

**Comment by NachoSoto** [comment]
It appears fetching offerings from load shedder is currently not working.

**Review by tonidero — APPROVED** [review]

**Comment by tonidero** [comment]
Oh, looks like CI considered this PR a release 😬

**Comment by NachoSoto** [comment]
Oh oops right, let me rename the branch.

**Pull request closed without merging**